plugins/thunderbolt/fu-thunderbolt-image.c

```diff
@@ -55,7 +59,9 @@ get_hw_info (guint16 id)
 		{ 0x15DA, 3, 1 }, /* AR-C 2C */
 
 		{ 0x15E7, 3, 1 }, /* TR 2C */
+		{ 0x15E8, 3, 1 }, /* TR 2C */
```

Comment:
For AR, for example, we use only the bridge ID. Any reason to use also the NHI ID in TR case?

Reply:
That's a good question. @finikorg what was the idea here?

Reply:
It turns out the Windows side is also using the same TR IDs so I think we can keep these.